core-java/src/main/java/eu/neverblink/jelly/core/JellyConverterFactory.java

```diff
@@ -62,7 +62,7 @@ public final ProtoDecoder<TNode, TDatatype> triplesDecoder(
      * Create a new QuadsDecoder.
      * @param supportedOptions maximum supported options for the decoder. If not provided, this.defaultSupportedOptions
      *                         will be used. If you want to modify this (e.g., to specify an expected logical stream
-     *                         type), you should always use this.defaultSupportedOptions.toBuilder().setXxx.build();.
+     *                         type), you should always use this.defaultSupportedOptions.clone().setXxx.build();.
```

Comment:
.build() is not requited anymore

Reply:
oui, good catch, removed that here and elsewhere.